### PR TITLE
W 10801489 improve branded emails 

### PR DESCRIPTION
**Pull request by JennyHajee**

### CODE REVIEW

**Review by nicolasgnr — DISMISSED** [review]

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/branded-emails.adoc [hunk 1]
```diff
@@ -101,11 +101,12 @@ The JSON Web Token (JWT) authentication mechanism creates Anypoint user records
 . Log in to your API community as an administrative user.
 . In *Access Management*, click *Users*.
 . Click the impersonated Anypoint user with the email address of the Salesforce administrator.
+. Add the Organization Administrator role to the user on each organization and suborganizations.
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/branded-emails.adoc]: I'd suggest "... each organization and its suborganizations" or "...each organization and suborganization", depending on your meaning

File: modules/ROOT/pages/branded-emails.adoc [hunk 2]
```diff
@@ -101,11 +101,12 @@ The JSON Web Token (JWT) authentication mechanism creates Anypoint user records
...
 . Click *Permissions*, click *API Manager*, and for each environment, open the permissions list and enable the *Manage Contracts* permission.
 . In the public side of the community, request access to an API with manual approval.
 . Log in to Anypoint and approve the requested API contract.
 +
 This creates the Anypoint user named *Automated Process*, which works with the Salesforce user named *Automated Process* to process triggers.
-. Grant the Anypoint user named *Automated Process* the *Organization Administrator* role xref:access-management::roles.adoc[using Anypoint Access Management].
+. Grant the Anypoint user named *Automated Process* the *Organization Administrator* role in the Main Organization xref:access-management::roles.adoc[using Anypoint Access Management].
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/branded-emails.adoc]: Should Main Organization be capitalized? I would expect "main organization"

**Review by bjoson-mulesoft — DISMISSED** [review]

File: modules/ROOT/pages/branded-emails.adoc (hunk 1 above)
Comment (JennyHajee) [reply modules/ROOT/pages/branded-emails.adoc]: Thanks, I will update this.

File: modules/ROOT/pages/branded-emails.adoc (hunk 2 above)
Comment (JennyHajee) [reply modules/ROOT/pages/branded-emails.adoc]: Yes, you are right. I will make this change as well.

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**